docs/protocol-specification/protocol-data-types.md

```diff
@@ -163,12 +163,7 @@ is used to indicate the name of the quantity and the JSON\* type.
 ]
 ```
 	    </td>
-            <td>See section <a href="../data-transmission-persistence/index.html#data-transmission-persistence">§</a>. The length is stored in hexadecimal format fully specified as 4 hex characters. A message longer than 0xFFFF bytes is stored as a 32-bit hexadecimal number fully specified as 8 hex characters.</td>
-        </tr>
-        <tr>
-            <td></td>
-            <td>Length((String containing hex),</td>
-            <td></td>
+            <td>See section <a href="../data-transmission-persistence/index.html#data-transmission-persistence">§</a>. The length is stored in hexadecimal format fully specified as 4 hex characters. A message longer than 0xFFFF bytes is stored as a 32-bit hexadecimal number fully specified as 8 hex characters. Implementations should store hex strings using upper case letters.Implementations should parse hex strings whether they have upper case or lower case letters in them.This message header need not be transmitted in some protocols.</td>
```

Comment:
Add a space after full stop.

letters.Implementations => letters. Implementations
them.This => them. This

Reply:
Updated.